package.json

```
@@ -20,7 +20,9 @@
     "build:watch": "lerna run --scope @cwds/* --parallel build:watch",
     "dev": "yarn build:watch",
     "storybook": "start-storybook -p 9001 -c .storybook",
-    "release": "lerna publish --conventional-commits --force-publish"
+    "release": "lerna publish --conventional-commits --force-publish",
+    "lint": "eslint --ext .jsx --ext .js libs/",
+    "fix-lint": "eslint --ext .jsx --ext .js --fix libs/"
```

can we follow the `general:specific:micro` convention (piggybacking add'l args when possible):

```suggestion
    "lint:fix": "yarn lint --fix"
```

---

👍